[14.0][ADD]sale_purchase_requisition (#2430)
OCA/sale-workflow: sale_purchase_requisition/readme/USAGE.rst

To use this module, you need to:

#. Install the module

> #. Automatically after save the new quotation apears a new button that we permit to create a new purchase agreements passing automatically all lines of your quotation.

Reviewer: ```suggestion
#. Automatically after saving the new quotation, there will apear a new button that allows to create a new purchase agreement passing automatically all lines of your quotation.
```

> #. If you add a value in the purchase_agent field this value will pass in the purchase agreement what you are creating.

Reviewer: ```suggestion
#. If you add a value in the purchase agent field, this value will pass to the purchase agreement what you are creating.
```
